test/local_sensitivity/sde.jl

```diff
@@ -37,13 +37,11 @@ p2 = [1.01,0.87]
   σ_oop_linear(u,p,t) = p[2]*u
 
   function f_oop_linear(u::Tracker.TrackedArray,p,t)
-    dx = p[1]*u[1]
-    Tracker.collect([dx])
+    p[1].*u
```

Comment:
@frankschae this decreases it a ton and now the SO goes away. Not really the best long term solution, but it's at least an improvement we should do.

Reply:
Oh, I didn't know that this works. I always used Tracker with the collect so far.